r/R/dplyr-functions.R

```diff
@@ -398,6 +398,22 @@ nse_funcs$str_split <- function(string, pattern, n = Inf, simplify = FALSE) {
   )
 }
 
+nse_funcs$pmin <- function(..., na.rm = FALSE) {
+  Expression$create(
+    "element_wise_min",
+    ...,
+    options = list(skip_nulls = na.rm)
+  )
+}
+
+nse_funcs$pmax <- function(..., na.rm = FALSE) {
+  Expression$create(
+    "element_wise_max",
```

Comment:
They're called `min_element_wise` and `max_element_wise` actually ;-)

Reply:
yea... last nigth it worked